tests/tablespaces/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG PGVERSION=10
+
+FROM pg_auto_failover_test:pg${PGVERSION} as test
+
+USER root
+RUN mkdir -p /tmp/extended_a && chown -R docker /tmp/extended_a
+RUN mkdir -p /tmp/extended_b && chown -R docker /tmp/extended_b
+RUN mkdir -p /tmp/extended_c && chown -R docker /tmp/extended_c
+RUN mkdir -p /tmp/extended_c && chown -R docker /tmp/extended_c
+RUN mkdir -p /var/lib/postgres && chown -R docker /var/lib/postgres
```

Comment:
Can we install the table spaces not in /tmp?

I am thinking we could use /var/lib/postgres/tablespaces/{a,b,c,d} ; given that PGDATA is going to be found in /var/lib/postgres/10/main for PGVERSION 10. What do you think?

(Using /tmp triggers too much overhead thinking about what kind of mount point that is, file system, sticky bits on the directory, etc, and how that impacts the test, do we want to test those elements etc).

Reply:
I'll use something that isn't `tmp` but I want to demonstrate that they are external volume mounts (or at least imply it) I will review! :)